Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -1,7 +1,15 @@
 namespace gdjs {
+  declare var jsSHA: any;
```

Comment:
It doesn't seem that hard to type. At least even if not describing the whole library then its usage here:
```suggestion
  declare const jsSHA: (algorithm: "SHA-256", type: "TEXT", options: {encoding: "UTF8"}) => { getHash: (format: "B64") => string, update: (payload: string) => void, };
```

Reply:
Actually I think the library has some d.ts that we can include: https://github.com/Caligatio/jsSHA/blob/master/dist/sha256.d.ts

Reply:
I'll see how to include it